src/views/virtualmachines/details/tabs/configuration/scripts/ScriptsTab.tsx

```diff
@@ -101,6 +103,11 @@ const ScriptsTab: React.FC<VirtualMachineScriptPageProps> = ({ obj: vm }) => {
               }
               descriptionHeader={t('Authorized SSH Key')}
               isEdit={canUpdateVM}
+              label={
+                <span className="pf-u-pl-xs pf-u-ml-xs">
```

Comment:
I can see the` <span>` element around `OSLabel` component everywhere in this PR. What is that span for? Is it really needed? Can't we just add the styling we need to the OSLabel by defining the css for that component? Or if we do need that span, we could move it to the OSLabel, too. WDYT? Thanks!